<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Query</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupWhat">
           <property name="title">
            <string>What</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QRadioButton" name="radioSelectLinkedResources">
              <property name="text">
               <string>&amp;Linked resources</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioSelectUsedResources">
              <property name="text">
               <string>&amp;Used resources</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioSelectAllResources">
              <property name="text">
               <string>Combined (no&amp;t implemented)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupWhat_2">
           <property name="title">
            <string>Ordering</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QRadioButton" name="radioOrderHighScoredFirst">
              <property name="text">
               <string>Hi&amp;gh score</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioOrderRecentlyUsedFirst">
              <property name="text">
               <string>Recentl&amp;y used resources</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioOrderRecentlyCreatedFirst">
              <property name="text">
               <string>Rece&amp;ntly created first</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioOrderByUrl">
              <property name="text">
               <string>By url</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioOrderByTitle">
              <property name="text">
               <string>By t&amp;itle</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <property name="horizontalSpacing">
            <number>8</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="labelActivity">
             <property name="text">
              <string>Activity</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="comboActivity">
             <property name="editable">
              <bool>true</bool>
             </property>
             <property name="currentText">
              <string>:current</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelAgent">
             <property name="text">
              <string>Agent</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="textAgent">
             <property name="text">
              <string>:any</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelMimetype">
             <property name="text">
              <string>Mimetype</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="textMimetype">
             <property name="text">
              <string>:any</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="buttonUpdate">
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>176</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Logs</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QTextBrowser" name="textLog">
           <property name="lineWrapMode">
            <enum>QTextEdit::NoWrap</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QListView" name="viewResults">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
